src/FsSpreadsheet/FsColumn.fs

```diff
@@ -0,0 +1,136 @@
+/// </summary>
+/// <remarks>The FsCellsCollection must only cover 1 column!</remarks>
+/// <exception cref="System.Exception">if given FsCellsCollection has more than 1 column.</exception>
+type FsColumn (rangeAddress : FsRangeAddress, cells : FsCellsCollection)= 
+
+    inherit FsRangeBase(rangeAddress)
+
+    let cells = cells
+
+    new() = FsColumn (FsRangeAddress(FsAddress(0,0),FsAddress(0,0)),FsCellsCollection())
+
+    /// <summary>
+    /// Create an FsColumn from a given FsCellsCollection and an columnColumn.
+    /// </summary>
+    /// <remarks>The appropriate range of the cells (i.e. minimum colIndex and maximum colIndex) is derived from the FsCells with the matching rowIndex.</remarks>
+    new(index, (cells : FsCellsCollection)) = 
+        let getIndexBy (f : (FsCell -> int) -> seq<FsCell> -> FsCell) = 
+            match cells.GetCellsInColumn index |> Seq.length with
+            | 0 -> 1
+            | _ ->
+                (
+                    cells.GetCellsInColumn index 
+                    |> f (fun c -> c.Address.RowNumber)
+                ).Address.RowNumber
+        let minRowIndex = getIndexBy Seq.minBy
+        let maxRowIndex = getIndexBy Seq.maxBy
+        FsColumn (FsRangeAddress(FsAddress(minRowIndex, index),FsAddress(maxRowIndex, index)), cells)
+
+    interface IEnumerable<FsCell> with
+        member this.GetEnumerator() : System.Collections.Generic.IEnumerator<FsCell> = this.Cells.GetEnumerator()
+
+    interface IEnumerable with
+        member this.GetEnumerator() = (this :> IEnumerable<FsCell>).GetEnumerator() :> IEnumerator
+
+    // ----------
+    // PROPERTIES
+    // ----------
+
+    /// The associated FsCells.
+    member self.Cells = 
+        base.Cells(cells)
+
+    /// <summary>
+    /// The index of the FsColumn.
+    /// </summary>
+    member self.Index 
+        with get() = self.RangeAddress.FirstAddress.ColumnNumber
+        and set(i) = 
+            self.RangeAddress.FirstAddress.ColumnNumber <- i
+            self.RangeAddress.LastAddress.ColumnNumber <- i
+
+
+    // -------
+    // METHODS
+    // -------
+
+    /// <summary>
+    /// Creates a deep copy of this FsRow.
+    /// </summary>
+    member self.Copy() =
+        let ra = self.RangeAddress.Copy()
+        let cells = self.Cells |> Seq.map (fun c -> c.Copy())
+        let fcc = FsCellsCollection()
+        fcc.Add cells
+        FsColumn(ra, fcc)
+
+    /// <summary>
+    /// Returns a deep copy of a given FsColumn.
+    /// </summary>
+    static member copy (column : FsColumn) =
+        column.Copy()
+
+    /// <summary>
+    /// Returns the index of the given FsColumn.
+    /// </summary>
+    static member getIndex (column : FsColumn) = 
+        column.Index
+
+    /// <summary>
+    /// Returns the FsCell at rowIndex.
+    /// </summary>
+    member self.Cell(rowIndex) = 
+        base.Cell(FsAddress(rowIndex,1),cells)
```

Comment:
Maybe we need to add a comment because this is like the third or fourth time that I stumbled upon this hardcoded `1` for column index and every time I look up how the `Cell` method is implemented in FsRangeBase because I always think that it will return the first column no matter what though it is implemented differently in FsRangeBase. 😅

Also, this should get `[<Obsolete>]` attribute.

Reply:
[a43daef](https://github.com/CSBiology/FsSpreadsheet/pull/24/commits/a43daefdf58e528a5a85c1542726cfa2aba4cae7)